packages/mobile/src/redux/sagas.ts

```diff
@@ -38,13 +40,21 @@ function* loggerSaga() {
   }
 
   yield takeEvery('*', (action: AnyAction) => {
-    if (action && action.type && loggerBlacklist.includes(action.type)) {
```

Comment:
I believe the previous organization in this function was better:
1. It was more neat with less nesting
2. It skips JSON.stringify for blacklisted actions (which can be expensive for big actions like setting the recipient cache)